src/searchModal/advancedSearch.js

```diff
@@ -64,13 +65,20 @@ export default function advancedSearchFactory(config) {
          * @returns {Promise} - Request promise
          */
         updateCriteria: function (route) {
+            const addCriteriaSpan = $('.add-criteria-container a span');
+            addCriteriaSpan[0].classList.toggle('icon-add');
+            addCriteriaSpan[0].classList.toggle('icon-loop');
```

Comment:
Could you add the identifier to the element to be more precise? 
Could we don’t mix DOM native methods with jQuery and stick to the one?
For example: 
```suggestion
            const $criteriaIcon = $('.add-criteria-container a span').eq(0);
            $criteriaIcon.toggleClass('icon-add').toggleClass('icon-loop');
```
NOTE: Unfortunately the multiple toggle classes is not working in our current version